grails-app/controllers/org/broadinstitute/orsp/AuthenticatedController.groovy

```diff
@@ -322,6 +322,14 @@ class AuthenticatedController implements Interceptor, UserInfo {
         ] as JSON)
     }
 
+    def hasSession() {
+        if (getUser()) {
+            render(['session': true] as JSON)
+        } else {
+            render(['session': false] as JSON)
+        }
+    }
+
```

Comment:
Assuming getUser() returns either true or false, may use  : 

```groovy
render(['session': getUser()] as JSON) 
```

Reply:
It returns session or null, but I'll use `render(['session': getUser()!=null] as JSON) `